.vscode/scripts/run-vehicledatabroker.sh

```diff
@@ -20,47 +20,45 @@ echo "#######################################################"
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-DATABROKER_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databroker.version | tr -d '"')
-DATABROKER_PORT='55555'
-DATABROKER_GRPC_PORT='52001'
-sudo chown $(whoami) $HOME
-DATABROKER_ASSET_FOLDER="$ROOT_DIRECTORY/.vscode/scripts/assets/databroker/$DATABROKER_VERSION"
+DATABROKER_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databroker.image | tr -d '"')
+DATABROKER_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .databroker.version | tr -d '"')
 
-#Detect host environment (distinguish for Mac M1 processor)
-if [[ `uname -m` == 'aarch64' || `uname -m` == 'arm64' ]]; then
-    echo "Detected ARM architecture"
-    PROCESSOR="aarch64"
-    DATABROKER_BINARY_NAME="databroker-arm64.tar.gz"
-    DATABROKER_EXEC_PATH="$DATABROKER_ASSET_FOLDER/$PROCESSOR/databroker"
-else
-    echo "Detected x86_64 architecture"
-    PROCESSOR="x86_64"
-    DATABROKER_BINARY_NAME='databroker-amd64.tar.gz'
-    DATABROKER_EXEC_PATH="$DATABROKER_ASSET_FOLDER/$PROCESSOR/databroker"
-fi
-
-if [[ ! -f "$DATABROKER_EXEC_PATH/databroker" ]]
+RUNNING_CONTAINER=$(docker ps | grep "$DATABROKER_IMAGE" | awk '{ print $1 }')
```

Comment:
Please add Help() function to explain the parameters

Maybe we need to add verification in front to make sure that parameter is provided and in case of failure - clear error message shall be shown